Documentation/Storage-Configuration/Shared-Filesystem-CephFS/filesystem-mirroring.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Filesystem Mirroring Overview
+---
+
+Ceph filesystem mirroring is a process of asynchronous replication of snapshots to a remote CephFS file system. Snapshots are synchronized by mirroring snapshot data followed by creating a snapshot with the same name (for a given directory on the remote file system) as the snapshot being synchronized. It is generally useful when planning for Disaster Recovery. Mirroring is for clusters that are geographically distributed and stretching a single cluster is not possible due to high latencies.
+
+## Prerequisites
+
+This guide assumes you have created a Rook cluster as explained in the main [quickstart guide](../../Getting-Started/quickstart.md)
+
+## Create the Filesystem with Mirroring enabled
+
+The following will enable mirroring on the filesystem:
+
+```yaml
+apiVersion: ceph.rook.io/v1
+kind: CephFilesystem
+metadata:
+  name: myfs
+  namespace: rook-ceph
+spec:
+  metadataPool:
+    failureDomain: host
+    replicated:
+      size: 3
+  dataPools:
+    - name: replicated
+      failureDomain: host
+      replicated:
+        size: 3
+  preserveFilesystemOnDelete: true
+  metadataServer:
+    activeCount: 1
+    activeStandby: true
+  mirroring:
+    enabled: true
+    # list of Kubernetes Secrets containing the peer token
+    # for more details see: https://docs.ceph.com/en/latest/dev/cephfs-mirroring/#bootstrap-peers
+    # Add the secret name if it already exists else specify the empty list here.
+    peers:
+      secretNames:
+        #- secondary-cluster-peer
+    # specify the schedule(s) on which snapshots should be taken
+    # see the official syntax here https://docs.ceph.com/en/latest/cephfs/snap-schedule/#add-and-remove-schedules
+    snapshotSchedules:
+      - path: /
+        interval: 24h # daily snapshots
+        # The startTime should be mentioned in the format YYYY-MM-DDTHH:MM:SS
+        # If startTime is not specified, then by default the start time is considered as midnight UTC.
+        # see usage here https://docs.ceph.com/en/latest/cephfs/snap-schedule/#usage
+        # startTime: 2022-07-15T11:55:00
+    # manage retention policies
+    # see syntax duration here https://docs.ceph.com/en/latest/cephfs/snap-schedule/#add-and-remove-retention-policies
+    snapshotRetention:
+      - path: /
+        duration: "h 24"
+```
+
+## Create the cephfs-mirror daemon
+
+On the source storage cluster, deploy the `cephfs-mirror`  daemon by referring to [Filesystem Mirror CRD](../../CRDs/Shared-Filesystem/ceph-fs-mirror-crd.md).
```

Comment:
How about showing an example from the filesystem-mirror.yaml example and instruct them to create it? Then we can link to the CRD doc for more details. This is the pattern generally seen for creating CRs instead of just referring them to the CRD doc.